modules/upgrade/pages/server-minor-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,59 @@
+[server-minor-upgrade]
+= Server - Minor Upgrade
+
+Several times the year, the {productname} team releases minor upgrades of the {productname} Server.
+These updates offer improved or completely new features.
+
+[IMPORTANT]
+====
+For more information, see the release notes at https://www.uyuni-project.org/pages/stable-version.html.
+Reading the release notes is critical.
+Sometimes additional manual steps are required.
+====
+
+Installing such minor upgrades is similar to installing OS security and regular package updates.
+// It is recommended to install them in due time.
+// In general, these steps are required:
+
+.Procedure: Updating Packages on the {productname} Server
+
+By default, several update repositories are configured and enabled for the {productname} Server.
+New and updated packages will become available automatically.
+
+. On the {productname} Server, at the command prompt, as root, stop the spacewalk services:
++
+----
+spacewalk-service stop
+----
+. List available patches:
++
+----
+zypper list-patches
+----
+. Apply all available patches:
++
+----
+zypper patch
+----
+
+. Restart the spacewalk services:
++
+----
+spacewalk-service start
+----
++
+[NOTE]
+====
+Starting with {productname} 4.1, [command]``spacewalk-schema-upgrade`` is not needed anymore.
+The schema upgrade will be run during [command]``spacewalk-service start`` automatically.
+====
+
+[IMPORTANT]
+====
+Services affected by a package update are not automatically restarted after an update.
+You need to restart these services manually to avoid potential failures.
+Use [command]``zypper ps`` to check for applications that are using old code and require restarting.
+====
+
+Reboot the server if a patch update recommends rebooting.
```

Comment:
I think this belongs inside the admon?

Reply:
I'll move the reboot advise into the last step of the proc